scripts/qa_crowbarsetup.sh

```diff
@@ -1120,6 +1151,10 @@ EOF
     fi
 
     if [[ $want_ironic = 1 ]] ; then
+        local ironic_bc="${net_ironic}${ip_sep}255"
+        if (( $want_ipv6 > 0 )); then
+            ironic_bc=''
```

Comment:
Oh. No broadcast if IPv6. That is incredibly unclear how this is currently laid out.